doc/flatpak-mask.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
+    "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+
+<refentry id="flatpak-mask">
+
+    <refentryinfo>
+        <title>flatpak mask</title>
+        <productname>flatpak</productname>
+
+        <authorgroup>
+            <author>
+                <contrib>Developer</contrib>
+                <firstname>Alexander</firstname>
+                <surname>Larsson</surname>
+                <email>alexl@redhat.com</email>
+            </author>
+        </authorgroup>
+    </refentryinfo>
+
+    <refmeta>
+        <refentrytitle>flatpak mask</refentrytitle>
+        <manvolnum>1</manvolnum>
+    </refmeta>
+
+    <refnamediv>
+        <refname>flatpak-mask</refname>
+        <refpurpose>Mask out updates and automatic installation</refpurpose>
+    </refnamediv>
+
+    <refsynopsisdiv>
+            <cmdsynopsis>
+                <command>flatpak mask</command>
+                <arg choice="opt" rep="repeat">OPTION</arg>
+                <arg choice="plain" rep="repeat">PATTERN</arg>
+            </cmdsynopsis>
+    </refsynopsisdiv>
+
+    <refsect1>
+        <title>Description</title>
+
+        <para>
+          Flatpak maintains a list of patterns that that define which refs are masked.
+          A masked ref will never be updated or automatically installed (for example an
+          masked extension marked auto-download will not be downloaded). You can still
+          manually install such refs, but once they are installed the version will be pinned.
+        </para>
+        <para>
+          The patterns are just a partial ref, with the * character matching anything
+          within that part of the ref. Here are some example patterns:
+<programlisting>
+org.some.App
+org.some.App//unstable
+app/org.domain.*
+org.some.App/arm
+</programlisting>
+        </para>
+        <para>
+          To list the current set of masks, run this command without any patterns.
+        </para>
+    </refsect1>
+
+    <refsect1>
+        <title>Options</title>
+
+        <para>The following options are understood:</para>
+
+        <variablelist>
+            <varlistentry>
+                <term><option>-h</option></term>
+                <term><option>--help</option></term>
+
+                <listitem><para>
+                    Show help options and exit.
+                </para></listitem>
+            </varlistentry>
+
```

Comment:
You miss --version here, I think have that everywhere else.

Reply:
Really? I only see it in flatpak.xml

Reply:
Sorry, i meant --verbose

Reply:
But --verbose is there.